AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/Migrations/20230227202735_Permissions.Designer.cs

```diff
@@ -0,0 +1,174 @@
+                });
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.Setting", b =>
+                {
+                    b.Property<string>("SettingGroupId")
+                        .HasMaxLength(36)
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<int>("Version")
+                        .HasColumnType("int");
+
+                    b.Property<DateTimeOffset>("CreatedAt")
+                        .HasColumnType("datetime(6)");
+
+                    b.Property<string>("CreatedBy")
+                        .IsRequired()
+                        .HasMaxLength(36)
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<string>("Input")
+                        .IsRequired()
+                        .HasColumnType("longtext");
+
+                    b.Property<bool>("IsCurrent")
+                        .HasColumnType("tinyint(1)");
+
+                    b.Property<DateTime>("LastUpdatedAt")
+                        .IsConcurrencyToken()
+                        .ValueGeneratedOnAddOrUpdate()
+                        .HasColumnType("timestamp(6)");
+
+                    b.HasKey("SettingGroupId", "Version");
+
+                    b.HasIndex("IsCurrent");
+
+                    b.HasIndex("SettingGroupId");
+
+                    b.ToTable("Settings");
+                });
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.SettingGroup", b =>
+                {
+                    b.Property<string>("SettingGroupId")
+                        .HasColumnType("varchar(255)");
+
+                    b.Property<string>("CreatedBy")
+                        .IsRequired()
+                        .HasColumnType("longtext");
+
+                    b.Property<DateTime>("LastUpdatedAt")
+                        .IsConcurrencyToken()
+                        .ValueGeneratedOnAddOrUpdate()
+                        .HasColumnType("timestamp(6)");
+
+                    b.HasKey("SettingGroupId");
+
+                    b.ToTable("SettingGroups");
+                });
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.User", b =>
+                {
+                    b.Property<string>("UserId")
+                        .HasMaxLength(36)
+                        .HasColumnType("varchar(36)");
+
+                    b.Property<string>("Email")
+                        .IsRequired()
+                        .HasMaxLength(50)
+                        .HasColumnType("varchar(50)");
+
+                    b.Property<string>("Password")
+                        .IsRequired()
+                        .HasMaxLength(36)
+                        .HasColumnType("varchar(36)");
+
+                    b.HasKey("UserId");
+
+                    b.ToTable("Users");
+                });
+
+            modelBuilder.Entity("AppSettingsManagerApi.Infrastructure.MySql.Permission", b =>
```

Comment:
what is the difference between this and the previous `modelBuilder.Entity "AppSettingsManagerApi.Infrastructure.MySql.Permission"` in Line 24

Reply:
Yeah they're not that different, you can ignore them for the most part. If anything the sql file that gets added when we add migrations would probably be more useful to take a look at but it's also auto-generated.

Everything in these files you can understand if you just understand the SettingsContext.cs file